src/main/java/com/crud/library/domain/BorrowedBookDto.java

```diff
@@ -0,0 +1,25 @@
+package com.crud.library.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class BorrowedBookDto {
+    private long id;
```

Comment:
Jak Ci się chce to zacznij używać `UUID` jako idików :) częściej się je spotyka w produkcyjnym kodzie, bo wymuszają unikalność. https://phauer.com/2016/uuids-hibernate-mysql/